README.md

```diff
@@ -1,18 +1,70 @@
-Longhorn Upgrade Responder
+Upgrade Responder
 ========
 
-## Building
+## Overview
+Upgrade Responder enables developers to know how many users are using their applications. 
+This project is GDPR compliant since it doesn't store any information that can identify the user, not even IP address.
 
-`make`
+For example, longhorn use Upgrade Responder project to create the public metric dashboard at [https://metrics.longhorn.io/](https://metrics.longhorn.io/)
 
+## Usage
+The overall idea is that application developers modify their application to periodically send requests that contain current app version and Kubernetes version to the Upgrade Responder server.
+The Upgrade Responder will extract that information and store it into InfluxDB.
+The Upgrade Responder will also return a response that contains the latest version of the application.
+A Grafana dashboard will pull the measurements out of InfluxDB and draw statical graphs.
+The steps are detailed below:
 
-## Running
 
-`./bin/upgrade-responder --debug start --upgrade-response-config /etc/upgrade-responder/upgrade-response.json --influxdb-url http://localhost:8086`
+### 1. Running Upgrade Responder server
+Use the image `longhornio/upgrade-responder` with the starting command:
```

Comment:
Don't need to mention the image here.

Add a section `Deploy with Kubernetes`.

Reply:
We have a chart at https://github.com/rancher/eio-charts/tree/master/charts/longhorn-upgrade-responder . and https://github.com/rancher/eio-charts/tree/master/charts/harvester-upgrade-responder/0.2.0

We can add it here to help with the deployment in Kubernetes.